Modules/Filtering/LabelMap/test/itkLabelImageToStatisticsLabelMapFilterTest1.cxx

```diff
@@ -109,19 +109,34 @@ itkLabelImageToStatisticsLabelMapFilterTest1(int argc, char * argv[])
   i2l->SetNumberOfBins(numberOfBins);
   ITK_TEST_SET_GET_VALUE(numberOfBins, i2l->GetNumberOfBins());
 
-  using L2IType = itk::LabelMapToLabelImageFilter<I2LType::OutputImageType, ImageType>;
-  L2IType::Pointer         l2i = L2IType::New();
-  itk::SimpleFilterWatcher watcher2(l2i);
+  using L2IType = itk::LabelMapToLabelImageFilter<typename I2LType::OutputImageType, ImageType>;
+  typename L2IType::Pointer l2i = L2IType::New();
+  itk::SimpleFilterWatcher  watcher2(l2i);
 
   l2i->SetInput(i2l->GetOutput());
 
   using WriterType = itk::ImageFileWriter<ImageType>;
-  WriterType::Pointer writer = WriterType::New();
+  typename WriterType::Pointer writer = WriterType::New();
   writer->SetInput(l2i->GetOutput());
   writer->SetFileName(argv[3]);
   writer->UseCompressionOn();
 
   ITK_TRY_EXPECT_NO_EXCEPTION(writer->Update());
 
+  // testing mean and median
+  using OutputImageType = typename I2LType::OutputImageType;
+  using PixelType = typename OutputImageType::PixelType;
+  using LabelVectorType = typename OutputImageType::LabelVectorType;
+  ITK_TRY_EXPECT_NO_EXCEPTION(i2l->Update());
+  const OutputImageType * const features = i2l->GetOutput();
+  const LabelVectorType         labels = features->GetLabels();
+  for (PixelType label : labels)
+  {
+    const double mean{ features->GetLabelObject(label)->GetMean() };
```

Comment:
Were mean and median different from 255 before this fix?

Reply:
No, the testing is behind; there wasn't a test at all for these functions.  Now there is, though it does not discern the distinction between the code before and after this pull request.

Reply:
The repro case looks like it needs only a little work to turn it into a Python test for this class.

Reply:
Very good, thank you.  I've just added `LabelImageToStatisticsLabelMapFilterTest2` and force-pushed.